eng/scripts/Language-Settings.ps1

```diff
@@ -152,4 +152,20 @@ function Publish-java-GithubIODocs ($DocLocation, $PublicArtifactLocation)
       }
     }
   }
+}
+
+function Get-java-CSVMetadata () {
+  $metadata = Get-CSVMetadata -MetadataUri $MetadataUri
```

Comment:
I was expecting that this extra filtering would happen in the Get-java-GitHubIOArtifacts.

Reply:
It is currently divided into chunks.
`Get-java-CVSMetadata` returns the table in csv file. We have one operation which reads com.azure if there are multiple packages found. 

`Get-java-GitHubIOArtifacts` returns the list of the artifact name under `java` in blob.

The common scripts (Generate-DocIndex) will execute the comparison logic between the returns from above two functions. (For every artifact from blob, find the right service name from csv table.) We return the toc required mapping in Generate-DocIndex scripts. Use another function to generating the actual yml/md files.